profiles/lib/destination.lua

```diff
@@ -0,0 +1,27 @@
+local Destination = {}
+
+function Destination.get_destination(way)
+  local destination = way:get_value_by_key("destination")
+  local destination_ref = way:get_value_by_key("destination:ref")
+
+  -- Assemble destination as: "A59: Düsseldorf, Köln"
+  --          destination:ref  ^    ^  destination
```

Comment:
This is starting to get ugly: we try to encode destinations into the street name as

    $name ($destination:ref: $destination)

in case we can not use the default of

    $name ($ref)